frontend/src/modules/individualDetails/reducers.ts

```diff
@@ -0,0 +1,102 @@
+import { WritableDraft } from "immer/dist/types/types-external";
+import { createNetworkActionTypes } from "../../utils/actions";
+import { AnyAction } from "redux";
+import { FetchedState } from "../common";
+import { FMSIndividual, FMSSample } from "../../models/fms_api_models";
+import produce from "immer";
+import { PagedItems } from "../../models/paged_items";
+import { createItemsByID } from "../../models/frontend_models";
+import { clearFiltersReducer, setFilterReducer } from "../../components/shared/WorkflowSamplesTable/FilterReducers";
+
+export const LIST_TABLE = createNetworkActionTypes('INDIVIDUAL_DETAILS.LIST_TABLE')
+export const SET_SORT_BY = 'INDIVIDUAL_DETAILS.SET_SORT_BY'
+export const SET_INDIVIDUAL_DETAILS_SAMPLES_FILTER = 'INDIVIDUAL_DETAILS.SET_INDIVIDUAL_SAMPLES_FILTER'
+export const CLEAR_FILTERS = "INDIVIDUAL_DETAILS.CLEAR_FILTERS"
+
+export interface Individual {
```

Comment:
There is already an Individual type defined in frontend_models.ts, so this could cause some confusion for the next developers to work in this code. I would give this type a unique name, such as IndividualDetails or IndividualWithSamples - anything unique.